### ISSUE TITLE
DSP dump error when dspfirm.cdc is dumped

### ISSUE THREAD

**Issue by sks316** [ErmanSayin/Themely#40]
In have my dspfirm.cdc dumped, but Themely started playing the theme's BGM and showed me the DSP error.

**Comment by ErmanSayin** [comment]
You sure the DSP dump is at "/3ds/dspfirm.cdc"?

**Comment by sks316** [comment]
Yes, I am entirely sure, otherwise it wouldn't start playing the BGM at all.On May 6, 2017 2:15 AM, Erman <notifications@github.com> wrote:You sure the DSP dump is at "/3ds/dspfirm.cdc"?
